### PR TITLE
Feat: SnowflakeOS

### PR DESCRIPTION
**Pull request by Malix-off**
[Would probably be better above nixbsd](https://github.com/nix-community/awesome-nix/pull/247)

### CODE REVIEW

**Comment by Malix-off** [comment]
[Would probably be better bellow SnowflakeOS](https://github.com/nix-community/awesome-nix/pull/248)

File: README.md [hunk 1]
```diff
@@ -294,6 +294,7 @@
 ## Distributions
 
 * [NixNG](https://github.com/nix-community/NixNG) - A GNU/Linux distribution similar to NixOS, defining difference is a focus on containers and lightweightness.
+* [SnowflakeOS](https://snowflakeos.org/) - A NixOS based Linux distribution focused on beginner friendliness and ease of use.
```
Comment (cyntheticfox) [inline README.md:297]:
```suggestion
* [SnowflakeOS](https://snowflakeos.org/) - A NixOS-based Linux distribution focused on beginner friendliness and ease of use.
```

I do have to wonder what a NixOS cringe distro would look like :sweatsmile:

Comment (cyntheticfox) [inline README.md:297]:
```suggestion
* [SnowflakeOS](https://snowflakeos.org/) - A NixOS-based Linux distribution focused on beginner friendliness and ease of use.
```

I do have to wonder what a NixOS cringe distro would look like :sweatsmile:

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Dont know why it left two line comments -- probably a GitHub android app bug.

Fix the minor punctuation thing and its good to merge IMO

**Review by cyntheticfox — APPROVED** [review]
Looks good with the edit!

**Pull request merged**